### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Dadadon**
Added contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -242,12 +244,37 @@ <h2 class="language">Skills</h2>
...
+        </p>
+        <form>
+            <label for="fname">Full name:</label>
+            <input id="fname" type="text" maxlength="30" name="full-name" placeholder="Yeremias" required class="input">
+
+            <label for="email">Email</label>
+            <input id="email" type="email" name="email-address" placeholder="natajayanj14@gmail.com" required class="input">
+
+            <label for="textarea">Message:</label>
+            <textarea id="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required class="input text-area"></textarea>
```
Comment (dasileker) [inline index.html]: - [ ] The `input` field  should have a `length` and should also be `required` see screenshot 👇
zzz

File: index.html [hunk 2]
```diff
@@ -242,12 +244,37 @@ <h2 class="language">Skills</h2>
...
+            <button class="btn" type="submit">Get in touch</button>
+        </form>
```
Comment (dasileker) [inline index.html]: - [ ] The form should have an `action` and method `POST` like the requirement says. see screenshot 👇
rr

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Dadadon** 

Good job so far!
You are almost there, always keep the good spirit and let's add some missing elements to move on!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Dadadon) [reply index.html]: - The input field does have a max length of 30 and the message input has a max length of 500. 

- Every field in the form is also required

- Email input type is email

**Review by AshakaE — APPROVED** [review]
### Hi @Dadadon  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the  required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**